Weather/Weather/Scenes/Forecast14day/_HeaderView/Forecast14dayHeaderView.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  Forecast14dayHeaderView.swift
+//  Weather
+//
+//  Created by minh duc on 3/6/19.
+//  Copyright © 2019 minhduc. All rights reserved.
+//
+
+import UIKit
+import Reusable
+
+final class Forecast14dayHeaderView: UIView, NibLoadable {
+    @IBOutlet private weak var mainView: UIView!
+    @IBOutlet private weak var dateLabel: UILabel!
+    @IBOutlet private weak var expandButon: UIButton!
+    @IBOutlet private weak var descriptionLabel: UILabel!
+    @IBOutlet private weak var weatherImageView: UIImageView!
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+    }
+    
+    func setContentView(with data: ForecastWeather) {
+        let weather = Weather(data.weather.icon)
+        let temperature = "\(data.minTemperature.celsius) / \(data.maxTemperature.celsius)"
+        weatherImageView.image = weather.largeImage
+        descriptionLabel.text = data.weather.description
+        dateLabel.text = data.datetime.fullDayOfWeek(data.timezone) + "\n\(temperature)"
+    }
+    
+    func configureSubview(_ target: BaseViewController,  selector: Selector, section: Int) {
+        expandButon.do {
+            $0.tag = section
+            $0.addTarget(target, action: selector, for: .touchUpInside)
+        }
+        
+        mainView.do {
```

Comment:
nếu chỉ làm 1 tác vụt hì ko cần dùng .do
```suggestion
        mainView.setCornerRadius(8, color: .clear)
```